git-cheatsheet/lang/en.json

```diff
@@ -106,6 +106,10 @@
             "cmd": "branch --track <new branch name> <remote>/<branch>",
             "docs": "Create a new local branch from a remote-tracking branch."
         },
+        "init x": {
+            "cmd": "init <dir>",
+            "docs": "Create a new local Git repository in <dir>. The repository starts with no commits."
```

Comment:
Can you add any translations? I have hesitated in adding new commands because of the existing translations, but I think it will be OK to add at this point, with AI and all.

Reply:
hmm.. I only know english, and danish... So won't be of much help here... :(